
float32 baro_alt_meter			# Altitude above MSL calculated from temperature compensated baro sensor data using an ISA corrected for sea level pressure SENS_BARO_QNH.
float32 baro_temp_celcius		# Temperature in degrees celsius
float32 baro_pressure_pa		# Absolute pressure in pascals

float32 rho						# air density